### PR TITLE
Pull Request

### PR DESCRIPTION
**Pull request by Tyorden**
New Pull Request

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1350,7 +1350,7 @@ Shashank Gupta, Shashi, shashvat, shashvatshahi1998, Siju, Siju Samuel,
 Snease-Abq, Spencer Schaber, sremedios, srinivasan.narayanamoorthy, Steve Lang,
 Steve Nesae, Sumesh Udayakumaran, Supriya Rao, Taylor Jakobson, Taylor Thornton,
 Ted Chang, ThisIsPIRI, Thomas Deegan, Thomas Hagebols, tianyapiaozi, Tim Zaman,
-tomguluson92, Tongxuan Liu, TungJerry, v1incent, Vagif, vcarpani, Vikram Tiwari,
+tomguluson92, Tongxuan Liu, TungJerry, Tyler Satchel Orden, v1incent, Vagif, vcarpani, Vikram Tiwari,
```
Comment (mihaimaruseac) [inline RELEASE.md:1353]: This is for an older release. You might want to add the github username on the part corresponding to the next release, at the beginning of the file. Although, the names listed here are for functional changes, not typo fixes.

File: tensorflow/java/src/main/java/org/tensorflow/Output.java [hunk 2]
```diff
@@ -16,7 +16,9 @@
 package org.tensorflow;
 
 import java.util.Objects;
-
+import java.lang.annotation.Retention; 
+import java.lang.annotation.RetentionPolicy; 
+import java.lang.reflect.Method;
```
Comment (mihaimaruseac) [inline tensorflow/java/src/main/java/org/tensorflow/Output.java:21]: Why these imports?

File: tensorflow/java/src/main/java/org/tensorflow/Session.java [hunk 3]
```diff
@@ -17,7 +17,9 @@
 
 import java.util.ArrayList;
 import java.util.List;
-
+import java.lang.annotation.Retention; 
+import java.lang.annotation.RetentionPolicy; 
+import java.lang.reflect.Method;
```
Comment (mihaimaruseac) [inline tensorflow/java/src/main/java/org/tensorflow/Session.java:22]: Why these imports?

File: tensorflow/java/src/main/java/org/tensorflow/Session.java [hunk 4]
```diff
@@ -31,11 +33,11 @@
  * try (Session s = new Session(graph)) {
  *   try (Tensor x = Tensor.create(2.0f);
  *       Tensor y = s.runner().feed("x", x).fetch("y").run().get(0)) {
- *       System.out.println(y.floatValue());  // Will print 6.0f
+ *       System.out.println(y.floatValue()); // Will print 6.0f
```
Comment (mihaimaruseac) [inline tensorflow/java/src/main/java/org/tensorflow/Session.java:36]: Coding style mandates 2 spaces before start of inline comments

File: tensorflow/java/src/main/java/org/tensorflow/Session.java [hunk 5]
```diff
@@ -31,11 +33,11 @@
...
  *   }
  *   try (Tensor x = Tensor.create(1.1f);
  *       Tensor y = s.runner().feed("x", x).fetch("y").run().get(0)) {
- *       System.out.println(y.floatValue());  // Will print 3.3f
+ *       System.out.println(y.floatValue()); // Will print 3.3f
```
Comment (mihaimaruseac) [inline tensorflow/java/src/main/java/org/tensorflow/Session.java:40]: Coding style mandates 2 spaces before start of inline comments

File: tensorflow/java/src/main/java/org/tensorflow/Shape.java [hunk 6]
```diff
@@ -22,12 +22,12 @@
 
   /** Create a Shape representing an unknown number of dimensions. */
   public static Shape unknown() {
-    return new Shape(null);
+   return new Shape(null);
```
Comment (mihaimaruseac) [inline tensorflow/java/src/main/java/org/tensorflow/Shape.java:25]: Coding style mandates 2 space indentation.

File: tensorflow/java/src/main/java/org/tensorflow/Shape.java [hunk 7]
```diff
@@ -22,12 +22,12 @@
...
   }
 
   /** Create a Shape representing a scalar value. */
   public static Shape scalar() {
-    return new Shape(new long[0]);
+   return new Shape(new long[0]);
```
Comment (mihaimaruseac) [inline tensorflow/java/src/main/java/org/tensorflow/Shape.java:30]: Coding style mandates 2 space indentation.

File: tensorflow/java/src/main/java/org/tensorflow/Shape.java [hunk 8]
```diff
@@ -74,22 +74,22 @@ public int numDimensions() {
    * @return The size of the requested dimension or -1 if it is unknown.
    */
   public long size(int i) {
-    return shape[i];
+   return shape[i];
```
Comment (mihaimaruseac) [inline tensorflow/java/src/main/java/org/tensorflow/Shape.java:77]: Coding style mandates 2 space indentation.

File: tensorflow/java/src/main/java/org/tensorflow/Shape.java [hunk 9]
```diff
@@ -74,22 +74,22 @@ public int numDimensions() {
...
   }
 
   @Override
   public int hashCode() {
-    return Arrays.hashCode(shape);
+   return Arrays.hashCode(shape);
```
Comment (mihaimaruseac) [inline tensorflow/java/src/main/java/org/tensorflow/Shape.java:82]: Coding style mandates 2 space indentation.

File: tensorflow/java/src/main/java/org/tensorflow/Shape.java [hunk 10]
```diff
@@ -99,14 +99,14 @@ public boolean equals(Object obj) {
   @Override
   public String toString() {
     if (shape == null) {
-      return "<unknown>";
+     return "<unknown>";
```
Comment (mihaimaruseac) [inline tensorflow/java/src/main/java/org/tensorflow/Shape.java:102]: Coding style mandates 2 space indentation.

File: tensorflow/java/src/main/java/org/tensorflow/Shape.java [hunk 11]
```diff
@@ -74,22 +74,22 @@ public int numDimensions() {
...
   @Override
   public boolean equals(Object obj) {
     if (this == obj) {
-      return true;
+     return true;
     }
 
     if (obj instanceof Shape && Arrays.equals(this.shape, ((Shape) obj).shape)) {
-      return !hasUnknownDimension();
+     return !hasUnknownDimension();
```
Comment (mihaimaruseac) [inline tensorflow/java/src/main/java/org/tensorflow/Shape.java:92]: Coding style mandates 2 space indentation.

File: tensorflow/java/src/main/java/org/tensorflow/Shape.java [hunk 12]
```diff
@@ -74,22 +74,22 @@ public int numDimensions() {
...
   public int hashCode() {
-    return Arrays.hashCode(shape);
+   return Arrays.hashCode(shape);
   }
 
   @Override
   public boolean equals(Object obj) {
     if (this == obj) {
-      return true;
+     return true;
```
Comment (mihaimaruseac) [inline tensorflow/java/src/main/java/org/tensorflow/Shape.java:88]: Coding style mandates 2 space indentation.

File: tensorflow/java/src/main/java/org/tensorflow/Shape.java [hunk 13]
```diff
@@ -99,14 +99,14 @@ public boolean equals(Object obj) {
...
-      return "<unknown>";
+     return "<unknown>";
     }
     return Arrays.toString(shape).replace("-1", "?");
   }
 
   // Package-private constructor.
   Shape(long[] shape) {
-    this.shape = shape;
+    this.shape  =  shape;
```
Comment (mihaimaruseac) [inline tensorflow/java/src/main/java/org/tensorflow/Shape.java:109]: Coding style mandates exactly one space around binary operators and assignments

File: tensorflow/python/tools/freeze_graph.py [hunk 14]
```diff
@@ -14,13 +14,13 @@
 # ==============================================================================
 r"""Converts checkpoint variables into Const ops in a standalone GraphDef file.
 
-This script is designed to take a GraphDef proto, a SaverDef proto, and a set of
+This script is designed to take a GraphDef proto, a SaverDef proto, as well as a set of
```
Comment (mihaimaruseac) [inline tensorflow/python/tools/freeze_graph.py:17]: Let's keep `and`

File: tensorflow/python/tools/freeze_graph.py [hunk 15]
```diff
@@ -14,13 +14,13 @@
...
 variable values stored in a checkpoint file, and output a GraphDef with all of
 the variable ops converted into const ops containing the values of the
 variables.
 
 It's useful to do this when we need to load a single file in C++, especially in
-environments like mobile or embedded where we may not have access to the
+mobile or embedded environments where we may not have access to the
```
Comment (mihaimaruseac) [inline tensorflow/python/tools/freeze_graph.py:23]: Should revert as the proposed changes eliminate a class of environments

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Comment by gbaned** [comment]
@Tyorden  Can you please check @mihaimaruseac's comments and keep us posted. Thanks!

**Comment by gbaned** [comment]
@Tyorden Any update on this PR? Please. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**